### ISSUE TITLE
Better information when a file is changed

### ISSUE THREAD

**Issue by karun012** [karun012/arion#23]
Right now when a file is changed Arion just says -

``` bash
/foo/bar/baz.hs changed
```

I think it should show the user more information like what tests are associated with the file, and also the module name instead of the absolute path.

``` bash
Module Baz changed

Baz is associated with these tests
BazSpec
FooSpec
BarSpec

...
```

**Comment by mwotton** [comment]
Hm. Isn't this just noise? Won't you see the tests themselves running?

**Comment by karun012** [comment]
I feel it's hard to say which tests actually ran. I don't know. Maybe it is
a little noisy.

**Comment by mwotton** [comment]
Yeah. Hm. Well, I guess from my zen-mode goals I shouldn't actually have to
know what's going to run - I just want the right stuff to happen :) Perhaps
the time to notify is actually when you are running the test, rather than
at the top? you're going to scroll to the failing test anyway.

**Comment by karun012** [comment]
Yeah. Now that you mention zen-mode I think we should roll this back. We'll see how zen-mode goes and decide if we want to put this back or not :)
